oak/server/grpc_client_node.cc

```diff
@@ -164,6 +165,22 @@ bool GrpcClientNode::HandleInvocation(MessageChannelReadHalf* invocation_channel
   if (!ok) {
     LOG(ERROR) << "Failed to finish gRPC method invocation";
   }
+  if (!status.ok()) {
+    // Final status includes an error, so pass it back on the response channel.
+    oak::GrpcResponse grpc_rsp;
+    grpc_rsp.set_last(true);
+    grpc_rsp.mutable_status()->set_code(status.error_code());
```

Comment:
Is `status` not assignable directly?

Reply:
They're different types: `grpc_rsp.mutable_status()` is `google::rpc::Status*` but `status` is a `grpc::Status`